docs/develop/understanding-the-javascript-api-for-office.md

```diff
@@ -28,31 +28,91 @@ For more details around the Office.js CDN, including how versioning and backward
 
 **Applies to:** All add-in types
 
-Office.js provides an initialization event which gets fired when the API is fully loaded and ready to begin interacting with the user. You can use the **initialize** event handler to implement common add-in initialization scenarios, such as prompting the user to select some cells in Excel, and then inserting a chart initialized with those selected values. You can also use the initialize event handler to initialize other custom logic for your add-in, such as establishing bindings, prompting for default add-in settings values, and so on.
+Your add-in will often have start-up logic to do things such as:
 
-At a minimum, the initialize event would look like the follow example:     
+- Check that the user's version of Office will support all the Office APIs that your code calls
```

Comment:
I think this list needs periods following each item.
https://worldready.cloudapp.net/StyleGuide/Read?id=2700